tests/core/converters/test_interpolate.cpp

```diff
@@ -26,14 +26,49 @@ TEST(Converters, ATenUpsampleNearest1dConvertsCorrectly) {
 
   auto trt_in = at::clone(in);
   params = trtorch::core::conversion::get_named_params(g->inputs(), {});
+
   auto trt_results = trtorch::tests::util::RunGraphEngine(g, params, {trt_in});
+  auto trt = trt_results[0].reshape(jit_results[0].sizes());
+  ASSERT_TRUE(trtorch::tests::util::almostEqual(jit_results[0], trt, 2e-6));
+
+  trt_results = trtorch::tests::util::RunGraphEngineDynamic(g, params, {trt_in});
```

Comment:
Not sure if it will be too much repeated code maybe we could build utility functions for these tests but it would be good to separate static and dynamic tests.